pyfar/dsp/dsp.py

```diff
@@ -1506,6 +1512,9 @@ def deconvolve(system_output, system_input, fft_length=None, **kwargs):
     if not system_output.sampling_rate == system_input.sampling_rate:
         raise ValueError("The two signals have different sampling rates!")
 
+    if freq_range is None:
+        freq_range = (0, system_input.sampling_rate)
```

Comment:
```suggestion
        freq_range = (0, system_input.sampling_rate/2)
```
The Nyquist limit is half the sampling rate.